pkg/cli/admin/upgrade/status/examples/4.14.1-degraded-mcp.yaml

```diff
@@ -0,0 +1,162 @@
+apiVersion: v1
```

Comment:
nit: Do we really need to all these examples that are unnecessarily increasing repo size?

Reply:
I agree the inputs grew a bit :( I do not expect us to see more testcases but I think we'll eventually need more inputs; I'll make sure we come up with some solution until then, maybe write a script that prunes the mock data from unnecessary members or something similar.